### ISSUE TITLE
python 3.7 invalid syntax

### ISSUE THREAD

**Issue by leonyu** [bittner/pyclean#4]
```
Traceback (most recent call last):
  File "/home/leonyu/.local/share/virtualenvs/log_clusterer-ORU4GgFM/bin/pyclean", line 6, in <module>
    from pyclean.pyclean import main
  File "/home/leonyu/.local/share/virtualenvs/log_clusterer-ORU4GgFM/lib/python3.7/site-packages/pyclean/pyclean.py", line 63
    except (IOError, OSError), e:
                             ^
SyntaxError: invalid syntax
```

**Comment by bittner** [comment]
Thanks for reporting!

This issue is due to the fact that the 3 scripts, `pyclean`, `py3clean`, `pypyclean`, are not cross-version compatible. (... and one of the reasons why this project exists!)

Please use `py3clean`, which should work on both Python 2 and 3.

**Comment by bittner** [comment]
Fixed via PR #7.

The code on the `master` branch now autodetects the Python version you're running.

I've also added usage instructions to the [README](https://github.com/bittner/pyclean#usage). Hope that helps!

**Comment by bittner** [comment]
I have released v2.0.0, which should now work on any platform supported by Python3.

Can you try whether it does what you expect?
